static/js/components/ObservingRunList.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import { observingRunTitle } from "./AssignmentForm";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    maxWidth: "22.5rem",
+    backgroundColor: theme.palette.background.paper,
+  },
+}));
+
+const SimpleList = ({ observingRuns }) => {
```

Comment:
Why define this separately instead of just including the relevant code in `ObservingRunList`? It'd read a lot easier to me if it were a single component

Reply:
Good point. I think I was writing this when I was first learning React. Crazy to think that was only like 3 or 4 weeks ago. Looking back on it now, it all seems very crude...

Reply:
(This is a port of #616)

Reply:
yeah, amazing how quickly you've become a React pro!

Reply:
Agree with @acrellin, but don't think it's a showstopper. I had the same feeling looking at my first React component on F that Stéfan somehow allowed to get merged...